README.md

```diff
@@ -71,4 +128,7 @@ class YourPushService: KlaviyoPushService() {
         KlaviyoPushService.handlePush(message.data, KlaviyoCustomerProperties())
     }
 }
-```
\ No newline at end of file
+```
+
+## Code Documentation
+Browse complete code documentation autogenerated with dokka [here](https://klaviyo.github.io/klaviyo-android-sdk/)
```

Comment:
The generated docs look great. @ndurell we should do something similar with docc. Once docc is setup may be github actions can generate them whenever something is merged to main and host it.

Reply:
Yup!